collect_app/src/androidTest/java/org/odk/collect/android/DaggerAndroidTest.java

```diff
@@ -0,0 +1,22 @@
+package org.odk.collect.android;
+
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.injection.AndroidTestComponent;
+import org.odk.collect.android.injection.DaggerAndroidTestComponent;
+
+import javax.annotation.OverridingMethodsMustInvokeSuper;
+
+public abstract class DaggerAndroidTest {
+
+    protected AndroidTestComponent androidTestComponent;
+
+    protected abstract void injectDependencies();
```

Comment:
A comment could also go here as well, just stating that the `androidTestComponent` has to be injected in the actual class with the dependencies and such or that's probably obvious 😅

Reply:
Sure